addons/maintenance/models/maintenance.py

```diff
@@ -234,7 +234,13 @@ def _create_new_request(self, date):
             'maintenance_team_id': self.maintenance_team_id.id,
             'duration': self.maintenance_duration,
             'company_id': self.company_id.id or self.env.company.id
-            })
+        }
+
+    def _create_new_request(self, date):
+        self.ensure_one()
+        vals = self._prepare_maintenance_request_vals(date)
+        maintenance_requests = self.env['maintenance.request'].create(vals)
+        return maintenance_requests
```

Comment:
@tranngocson1996 this looks better
```suggestion
        return self.env['maintenance.request'].create(vals)
```

Reply:
> @tranngocson1996 this looks better

@royleviindoo thank you bro :D